pkg/kp/pcstore/pcstoretest/fake_pcstore.go

```diff
@@ -56,3 +57,19 @@ func (p *FakePCStore) Delete(id fields.ID) error {
 	delete(p.podClusters, id)
 	return nil
 }
+
+func (p *FakePCStore) FindWhereLabeled(
```

Comment:
I like the idea of using a fake applicator, since there will be needs for a fake pcstore. Will save for a future change though.